README.md

```diff
@@ -42,9 +46,14 @@ DATABASE_PASSWORD=Welcome
 
 ## Updating the database schema
 
+First get hold of the IP address for your local database running in docker:
+```sh
+docker inspect -f '{{range.NetworkSettings.Networks}}{{.IPAddress}}{{end}}' postgres-local
+```
+
 The following command will deploy the liquibase config using the docker image when run from the root of the project.
-You will need to replace the value [YOUR IP ADDRESS] with the IP address for your database. If you are not running on the default port you may also need to update that in the connection string. Check that the username and password are also correct
+You will need to replace the value [YOUR POSTGRES CONTAINER’S IP ADDRESS] with the IP address for your database. If you are not running on the default port you may also need to update that in the connection string. Check that the username and password are also correct
 
 ```sh
-docker run -it --rm -v $(pwd)/db/changelog:/liquibase/changelog -v $(pwd)/db/sql:/liquibase/sql liquibase/liquibase  --url="jdbc:postgresql://[YOUR IP ADDRESS]:5432/callisto" --changeLogFile=changelog/db.changelog-main.yml --username=postgres --password=Welcome update
+docker run -it --rm -v $(pwd)/db/changelog:/liquibase/changelog -v $(pwd)/db/sql:/liquibase/sql liquibase/liquibase  --url="jdbc:postgresql://[YOUR POSTGRES CONTAINER’S IP ADDRESS]:5432/callisto" --changeLogFile=changelog/db.changelog-main.yml --username=postgres --password=Welcome update
```

Comment:
When I run this command, I use my own IP address which is different to my Postgres container's IP address I get from the command above and it seems to work for me

Reply:
No, my personal IP address for my laptop

Reply:
For the record, we decided to remove this particular readme change as we felt we didn’t understand it sufficiently and weren’t really interested in investing the time to do so at this stage